src/upload/_style.scss

```diff
@@ -6,4 +6,15 @@
     top: 0;
     left: 0;
     cursor: pointer;
-  }
\ No newline at end of file
+  }
+
+  @include media-breakpoint-up(md) {
+    .collapsible-card {
```

Comment:
I'd suggest creating another class for this so we can apply it to the status banner as well.  When I click the download button that banner is extending beyond the card which looks odd